src/CourseDashboard.cpp

```diff
@@ -82,3 +83,40 @@ void CourseDashboard::updateUser(User & user)
         break;
     }
 }
+
+void CourseDashboard::logIn()
+{
+    std::string login, password, loginKeyword, passwordKeyword;
+
+    std::cout << "UserName: "; std::cin >> loginKeyword;
+    std::cout << "Password: "; std::cin >> passwordKeyword;
+
+    std::fstream file;
+    file.open("plik.txt", std::ios::in);
+    //std::getline(file,login);
+    //std::getline(file,password);
```

Comment:
No comments should be in production code

Reply:
already in second commit this comments don't exist